javascript/javascript1/week2/fullNameFunction.js

```diff
@@ -0,0 +1,26 @@
+function getFullName(firstName, surName, useFormalName, gender = 'm')
+{   
+    if(firstName && surName)
+    {
+        let initial = (gender === 'm') ? 'Lord ':'Lady ';
+        let fullName = firstName + ' ' + surName;
+        return (useFormalName === true)? initial + fullName : fullName;
```

Comment:
consider if === true is necessary when handling a boolean value - the boolean value is already either true or false.